core/src/test/scala/integration/kafka/api/AdminClientIntegrationTest.scala

```diff
@@ -759,15 +758,17 @@ class AdminClientIntegrationTest extends IntegrationTestHarness with Logging {
       // Need to retry if leader is not available for the partition
       result = client.deleteRecords(Map(topicPartition -> RecordsToDelete.beforeOffset(0L)).asJava)
 
+      lowWatermark = Long.MinValue
       val future = result.lowWatermarks().get(topicPartition)
       try {
-        lowWatermark = future.get(1000L, TimeUnit.MILLISECONDS).lowWatermark()
+        lowWatermark = future.get().lowWatermark()
```

Comment:
I wonder if it's still useful to pass a timeout to `get()` to avoid blocking longer than the timeout passed to `waitUntil`. We could use `JTestUtils.DEFAULT_MAX_WAIT_MS` or manually set a specific timeout.

nit: no need for parenthesis after `lowWatermark`.

Reply:
Note that `requestTimeout` is 20 seconds, so I think `get` is OK.